spec/controllers/organizations_controller_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'spec_helper'
+
+describe OrganizationsController do
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:admin, is_admin: true) }
```

Comment:
Since you are using the `:admin` factory, the `is_admin: true` parameter is not necessary.
Either way, I think here we should be testing with a regular user, right?